src/test/scala/jtag/JtagShifterTest.scala

```diff
@@ -255,7 +255,7 @@ class JtagShifterSpec extends FlatSpec with ChainIOUtils {
       val d = Bool()
       val c = UInt(2.W)
 
-      override def cloneType: this.type = (new InnerBundle).asInstanceOf[this.type]
+//      override def cloneType: this.type = (new InnerBundle).asInstanceOf[this.type]
```

Comment:
Is there a reason this is commented instead of deleted?

Reply:
Yeah, I actually tried to do this as part of this update but it got to complicate. Seems like maybe this would be a candidate for ip-contributions

Reply:
Deleting it

Reply:
Yeah, I actually tried to refactor to ChiselTest  as part of this update but it got too complicated.